src/qt/forms/receivecoinsdialog.ui

```diff
@@ -213,6 +213,13 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="useMWEB">
```

Comment:
It doesn't look like the logic is tied in for this field yet. Are we missing an API to generate an MWEB address? I can add whatever you need.

Reply:
I thought mweb addresses need the one-sided code in place first?

Reply:
We have a GetAddress that we were going to use to encrypt partial transactions. See: https://github.com/ltc-mweb/libmw/blob/master/include/libmw/wallet.h#L98

It will change to return a stealth address once we merge the one-sided logic, but the API will stay the same. You can use that.